checkpoint/ddb/ddb.go

```diff
@@ -135,6 +127,18 @@ func (c *Checkpoint) Set(streamName, shardID, sequenceNumber string) error {
 	return nil
 }
 
+// ValidaCheckpoint validate the checkpoint table exits, shut down
```

Comment:
feel free to remove this if it doesn't add value; the original intention was to early-exit from the client setup if the table doesn't exist.

Reply:
I would prefer remove this. Does not look like it is adding much